src/command/dev/version_upgrade_message.rs

```diff
@@ -0,0 +1,18 @@
+#![warn(missing_docs)]
+use rover_std::Style;
+
+pub struct VersionUpgradeMessage {}
+
+impl VersionUpgradeMessage {
+    pub fn print() {
+        eprintln!();
```

Comment:
why the empty eprintln here?

Reply:
It's just to break it up a bit, it looks fine when it prints out :)

Reply:
@aaronArinder To give emphasis to the block, but I may be able to move this to line 11 now that I've switched from warnln! to eprintln! 

(If I put a \n at the beginning of line 11 when using infoln, it prints  `==> \n ** Notice`)

Here I've switched everything to eprintln! because I personally feel eyepoked by so many ==> s, and that's probably not a good reason.

I'm happy to switch these all back to infoln (or warnln) or observe any other formatting regularities.